package.json

```diff
@@ -23,7 +28,8 @@
     "prettier": "^2.2.0",
     "stylelint": "^13.8.0",
     "stylelint-config-prettier": "^8.0.2",
-    "stylelint-prettier": "^1.1.2"
+    "stylelint-prettier": "^1.1.2",
+    "typescript": "^4.1.4"
```

Comment:
What was TS added for? 
I'm would happily use it again 👀

Reply:
Yup, seems daft but it’s a **peerDependency** of a utility used by `eslint-plugin-jest`. Even though we're not using any of its Typescript features, it still chucks out this warning if you install without it:

```
npm WARN tsutils@3.20.0 requires a peer of typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta but none is installed. You must install peer dependencies yourself.
```

Comes from **tsutils**:

```
@coopdigital/foundations@ ./coop-frontend
└─┬ eslint-plugin-jest@24.1.3
  └─┬ @typescript-eslint/experimental-utils@4.15.0
    └─┬ @typescript-eslint/typescript-estree@4.15.0
      └── tsutils@3.20.0
```